src/ethereum_test_types/types.py

```diff
@@ -467,11 +467,14 @@ class AuthorizationTuple(AuthorizationTupleGeneric[HexNumber]):
 
     signer: EOA | None = None
     secret_key: Hash | None = None
+    yParity: HexNumber | None = None  # noqa: N815
```

Comment:
I would not add a new field, because these can be problematic sometimes.

I think we should move the `model_serializer` that #1286 added from `FixtureAuthorizationTuple` into `AuthorizationTupleGeneric`, because since now we need the same behavior in `FixtureAuthorizationTuple` and `AuthorizationTuple`, and both inherit `AuthorizationTupleGeneric`.